lambda-sqs-sls/example-pattern.json

```diff
@@ -0,0 +1,49 @@
+{
+  "title": "Lambda to SQS",
+  "description": "Publish events to SQS queue.",
+  "language": "Python",
+  "level": "200",
+  "framework": "Serverless",
+  "introBox": {
+    "headline": "Create a Lambda function that sends events to a SQS queue.",
+    "text": [
+      "The Serverless template deploys a SQS queue and a Lambda function that has permission to publish events to the queue."
+    ]
+  },
+  "gitHub": {
+    "template": {
+      "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/lambda-sqs-sls",
+      "templateURL": "serverless-patterns/lambda-sqs-sls"
+    }
+  },
+  "resources": {
+    "bullets": [
+      {
+        "text": "Using AWS Lambda with Amazon SQS",
+        "link": "https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html"
+      },
+      {
+        "text": "Choosing between messaging services for serverless applications.",
+        "link": "https://aws.amazon.com/blogs/compute/choosing-between-messaging-services-for-serverless-applications/"
+      }
+    ]
+  },
+  "deploy": {
+    "text": ["serverless deploy --verbose"]
+  },
+  "testing": {
+    "text": ["See the Github repo for detailed testing instructions."]
+  },
+  "cleanup": {
+    "text": [
+      "Delete the stack: <code>serverless remove --verbose</code>.",
+      "Confirm the stack has been deleted: aws cloudformation list-stacks --query \"StackSummaries[?contains(StackName,'STACK_NAME')].StackStatus\""
+    ]
+  },
+  "authors": [
+    {
+      "name": "Alex Nelson",
+      "bio": "Senior Solutions Architect, AWS"
+    }
+  ]
+}
```

Comment:
Think this pattern is already live https://serverlessland.com/patterns/lambda-sqs-python-sls

We can remove this folder.